nsdb-cluster/src/main/resources/application-common.conf

```diff
@@ -35,17 +35,18 @@ akka {
     }
   }
 
-  extensions = ["akka.cluster.pubsub.DistributedPubSub", "io.radicalbit.nsdb.cluster.extension.RemoteAddress"]
+  extensions = ["akka.cluster.pubsub.DistributedPubSub", "io.radicalbit.nsdb.cluster.extension.RemoteAddress", "akka.cluster.metrics.ClusterMetricsExtension"]
 
   cluster {
-    distributed-data {
-      durable {
-        keys = ["schema-cache-*", "metric-info-cache-*"]
-        lmdb.dir = ""
-        lmdb.write-behind-interval = 200 ms
+      metrics.collector.sample-interval = 10s
+      distributed-data {
+        durable {
+          keys = ["schema-cache-*", "metric-info-cache-*"]
+          lmdb.dir = ""
```

Comment:
should be empty this?

Reply:
Yes. It's correct. It's populated inside the application from a path taken from `nsdb.conf`

Maybe this needs a comment.